test/e2e/scheduling/preemption.go

```diff
@@ -118,21 +130,37 @@ var _ = SIGDescribe("SchedulerPreemption [Serial]", func() {
 			if i == 0 {
 				priorityName = lowPriorityClassName
 			}
-			pods[i] = createPausePod(f, pausePodConfig{
+			pods = append(pods, createPausePod(f, pausePodConfig{
```

Comment:
if the first node is skipped due to utilization, we never use `lowPriorityClassName`, where I think the intent is first pod created should use it regardless of which node it's scheduled to? maybe this should be checking against `len(pods)` instead

Reply:
I think the intent here was that the first pod should use it while all the other pods use medium priority. So in the case that the first pod doesn't get created, we do a bunch of checks to see if we're looking for a low priority a medium priority pod to be evicted.

But all that complicates the test imo. Could we just try to make 3 pods that are *all* low priority? @ravisantoshgudimetla, do you remember why we were doing it this way?